lib/common/client/middleware/request/camelcase.rb

```diff
@@ -10,7 +10,7 @@ def initialize(app)
           end
 
           def call(env)
-            env[:body] = camelcase(env[:body])
+            env[:body] = camelcase(env[:body]) unless env.request_headers['Content-Type'] == 'text/plain'
```

Comment:
All connections that use camelcase also use json, and this middleware assumes it's working with an Array or a Hash (json_has is the arg), so maybe it should be run only if it's 'application/json'? (unless we have VA services returning JSON with mislabelled content-types, which is entirely possible) 😄

Reply:
I thought about that -- but then I worried that there might be some weird hosts out there that don't respond well to "Content-Type" headers and we'd be excluding those.